geode-protobuf/src/test/java/org/apache/geode/protocol/RoundTripCacheConnectionJUnitTest.java

```diff
@@ -117,6 +119,7 @@ public void setup() throws Exception {
     boolean useSSL = testName.getMethodName().startsWith("useSSL_");
```

Comment:
I think that if we want to configure that these tests run with SSL, why don't we make the tests parameterized. Then it will test both SSL and non-SSL. Making a test SSL dependent on the naming convention is wrong.